deploy/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -83,6 +83,8 @@ spec:
           value: "{{ .Values.hostpathRequiresPrivileged }}"
         - name: ROOK_DISABLE_DEVICE_HOTPLUG
           value: "{{ .Values.disableDeviceHotplug }}"
+        - name: ROOK_DISCOVER_DAEMON_UDEV_BLACKLIST
```

Comment:
Where it is used in the project

Reply:
Thank you, this is indeed correct. I now verified by copy-pasting from the actual constant value...

Reply:
@JensErat LGTM after you push the updated constant name